src/messages/FileMessage.vue

```diff
@@ -25,12 +68,40 @@ export default {
     messageColors: {
       type: Object,
       required: true
+    },
+  },
+  methods:{
+    humanReadableFileSize(aSize) {
+      aSize = Math.abs(parseInt(aSize, 10));
+      var def = [[1, 'bytes'], [1024, 'kb'], [1024*1024, 'Mb'], [1024*1024*1024, 'Gb']];
+      for(var i=0; i<def.length; i++){
+        if(aSize<def[i][0]) return (aSize/def[i-1][0]).toFixed(2)+' '+def[i-1][1];
+      }
+    },
+    truncateFilename(filename, maxLength) {
```

Comment:
Same here, give a look to https://github.com/millerized/smart-truncate#usage :)

It seems that for file truncation, having the extension also visible would make sense
`Report_this_that-2019-2020-matteo-asbsdfhaocoif.pdf` -> `Report_this_that…coif.pdf`